machop/mase_components/activations/test/fixed_gelu_tb.py

```diff
@@ -0,0 +1,90 @@
+import random, os
+import numpy as np
+import cocotb
+from cocotb.triggers import Timer
+from mase_cocotb.runner import mase_runner
+
+import torch
+from torch import nn
+
+DATA_IN_0_PRECISION_1 = 8

```

Comment:
Should be defined under a class named `VerificationCase`

Reply:
VerificationCase is deprecated, we should use the Testbench class https://github.com/DeepWok/mase/blob/main/machop/mase_cocotb/testbench.py